sycl/test/extensions/composite_device/device_selector_test.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env ONEAPI_DEVICE_SELECTOR=ext_oneapi_level_zero:0 ZE_FLAT_DEVICE_HIERARCHY=COMBINED %t.out
```

Comment:
I'd expect that should be a e2e test instead. Probably the same for the previous test as well.